### DISCUSSION TITLE
Working with multiple agencies at once

### DISCUSSION THREAD

**Discussion by trees777** [VRSEN/agency-swarm#74] [Ideas]
Hi all,

I am looking into the idea of utilizing multiple agencies in one program. Where I am stuck right now is I would like for the CEO to pick one path, and stay on it until the user suggests otherwise. I thought passing CEO to one leader to their team would help but it gets lost with training documents. I also tried utilizing two different charts at once with buttons in gradio (errors) and same with a command line menu (settings.json error). I am a novice developer so this has been difficult but I can taste how close I am to getting this to work. Would love any ideas or suggestions!

Thank you.

**Comment by nicko-ai** [comment]
Creating several Agency instances is not supported due to current implementation limitations. In the new framework version, we’ll work on enabling a more scalable architecture.
